common/api/core-common.api.md

```diff
@@ -1991,6 +2060,8 @@ export class DisplayStyle3dSettings extends DisplayStyleSettings {
     set ambientOcclusionSettings(ao: AmbientOcclusion.Settings);
     // @internal
     applyOverrides(overrides: DisplayStyle3dSettingsProps): void;
+    get atmosphericScattering(): AtmosphericScattering.Settings;

```

Comment:
These settings might make more sense as part of `Environment`, since we cannot display an atmosphere unless we're also displaying a skybox.